### PR TITLE
Update Sanke.html

### PR DESCRIPTION
**Pull request by Juanes165**
changed the **textures** of the snake, solved bug with **growSnake**

### CODE REVIEW

File: Sanke.html [hunk 1]
```diff
@@ -186,6 +195,7 @@
 
             //executes itself in every f
             processing.onTic = function (world) {
+                console.log(world.snake)
```
Comment (srd4) [inline Sanke.html]: Console. log

File: Sanke.html [hunk 2]
```diff
@@ -202,23 +212,50 @@
                 return gotFood(movement(world));
             }
 
-            function drawSnake(list) {
+            /**
+             * Executes de function 'image' from processing as many times as the list length
```
Comment (srd4) [inline Sanke.html]: Documentation, proper english:
- as the length of the list
- Excecutes **the**
And:
- Image is not a function, is a method.

File: Sanke.html [hunk 3]
```diff
@@ -70,45 +71,48 @@
         //retorna true si la posicion de la cabeza es igual a la de la comid
```
Comment (Jpcqaa) [inline Sanke.html:71]: Falta una "a" al final de "comid" gonorrea disléxica

Comment (Juanes165) [reply Sanke.html:71]: eso lo tenes que corregir vos malparido, anda a joder a tu vieja

**Comment by Juanes165** [comment]
fixed score issue

File: Sanke.html [hunk 4]
```diff
@@ -120,8 +124,8 @@
         function sketchProc(processing) {
 
             processing.setup = function () {
-                processing.frameRate(20);
-                processing.size(40 * 20, 25 * 20);
+                processing.frameRate(15);
+                processing.size(40 * size, 25 * size);
```
Comment (Jpcqaa) [inline Sanke.html:127]: kehicistes aquí ?

Comment (Juanes165) [reply Sanke.html:127]: I changed the framerate and replaced the 20 number for the constant 'size'

File: Sanke.html (hunk 3 above)
Comment (Juanes165) [reply Sanke.html:71]: te faltó una "a" en tu comentario gonorrea dislexica

File: Sanke.html (hunk 4 above)
Comment (Jpcqaa) [reply Sanke.html:127]: Thank you.

**Review by srd4 — APPROVED** [review]
Very poor dev.

**Review by Stevie94 — APPROVED** [review]

**Pull request merged**